realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -308,7 +308,7 @@ private static void initializeRealm(Realm realm) {
     /**
      * Creates a Realm object for each object in a JSON array. This must be done within a transaction.
      * JSON properties with a null value will map to the default value for the data type in Realm and unknown properties
-     * will be ignored.
+     * will be ignored. If a Realm object property is not present in the JSON object the Realm object property will be {@code null}.
```

Comment:
Normally we use `field` instead of `property`, since that is more accurate in Java: http://stackoverflow.com/questions/10115588/what-is-the-difference-between-field-variable-attribute-and-property-in-java